content/docs/1.6.0/volumes-and-nodes/storage-class-parms.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Storage Class Settings
+weight: 3
+---
+
+## Overview
+
+Storage Class as a resource object has a number of settable parameters.  See also [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes).  
+Most of the ones in this list are specific to Longhorn's driver.  A few are common to all Kubernetes storage classes:
+
+### Default Class
+Set this Storage Class as the default for dynamic (Kubernetes-driven) volume creation.  
+Type: Boolean  
+Notes: This should only be true for one StorageClass object.  If not, either all volume creations will fail, or else one of the competing defaults will be chosen, and it may be hard to predict which one.  
+***(Which is it?  Give this a try.)***
```

Comment:
I'm sure you are planning on cleaning this up. Commenting here so we don't forget.